### ISSUE TITLE
auth broken

### ISSUE THREAD

**Issue by morethanhater** [onyxbits/Raccoon#60]
Can't authenticate though login&password are both correct
4.18
Simply doing as you stated at your blog isn't enough
There are no log-in attempts to allow.
https://raccoon.onyxbits.de/blog/bad-authentication-google-account-blocked/
```
java.lang.RuntimeException: org.apache.http.client.ClientProtocolException: BadAuthentication
	at de.onyxbits.raccoon.setup.LoginLogic.onDone(LoginLogic.java:138)
	at de.onyxbits.raccoon.setup.WizardWorker.done(WizardWorker.java:42)
	at javax.swing.SwingWorker$5.run(Unknown Source)
	at javax.swing.SwingWorker$DoSubmitAccumulativeRunnable.run(Unknown Source)
	at sun.swing.AccumulativeRunnable.run(Unknown Source)
	at javax.swing.SwingWorker$DoSubmitAccumulativeRunnable.actionPerformed(Unknown Source)
	at javax.swing.Timer.fireActionPerformed(Unknown Source)
	at javax.swing.Timer$DoPostEvent.run(Unknown Source)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
Caused by: org.apache.http.client.ClientProtocolException: BadAuthentication
	at com.akdeniz.googleplaycrawler.Identity.doPost(Identity.java:191)
	at com.akdeniz.googleplaycrawler.Identity.signIn(Identity.java:152)
	at com.akdeniz.googleplaycrawler.GooglePlayAPI.login(GooglePlayAPI.java:319)
	at de.onyxbits.raccoon.setup.LoginLogic.onDoInBackground(LoginLogic.java:88)
	at de.onyxbits.raccoon.setup.WizardWorker.doInBackground(WizardWorker.java:36)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

```

**Comment by onyxbits** [comment]
- What is the output of DummyDroid?
- Does this affect only Raccoon/DummyDroid? 
- If you can still login via Browser: does it work with all browsers or just Chrome?
- Can you log in with Android (this means you have to remove and add your account again)?


What can you tell me about the account?
- How old is the account?
- Do you have "allow less secure apps" enabled in your account settings?
- Do you have a telephone number registered to it (if so, is it a "real" telephone number or one from an "free SMS service")?
- From what networks do you usually log that account in? Did you recently switch ISP, move to another city, change your carrier, or did anything else that might result in Google seeing you logging in from a different IP subnet (e.g. use of proxies)?
- How do you use your account outside of Raccoon? Is it bound to a real device?

Other things
- Did anything "sketchy" happen from your IP address lately? For example, are you aware of any malware infections that might have resulted in your computer sending SPAM through Gmail?
- Do you run apps with a shady monetization  scheme? There's one in particular where app developers can earn money from making their apps the exit node of a VPN service (can't recall the name of the SDK "ninja something", I think). If one of their users pisses off Google (and lets face it, that's 90% of the reasons for using a VPN), the blame is on you.
- Are you aware of your ISP having high bot activity in your subnet (your ISP may be using Carrier grade NAT and is most certainly using dynamic IP addresses, so it is possible that something, someone else is doing is wrongly attributed to you).

**Comment by morethanhater** [comment]
while i can't access my dummydroid license because of another pc, that's what i can tell for sure:
1. Yes, i can login through any browser (vivaldi, opera, chrome, edge, firefox)
2. Yes, i can re-login at my Android phone and even iPhone, so account isn't blocked

And some more info:
I can't auth with any account, new, old, personal - doesn't matter

If you need specific info about the first account i did try to auth today:
1. It's more than 2 years old
2. Nope, "allow less secure apps" were disabled but enabling it didn't resolve the issue with account that i already used within raccoon and dummydroid
3. Yes, my second phone number is connected to it
4. Yes, i did switch ISP (work-home) but until today there were no problems because of that. These are my usual 2 ISP. And no, i didn't use any proxy.
5. Yes, this account is bound to real device. Even two of them.

And about other things:
1. Nope, nothing like this. I would be aware of it in a short time.
2. No, for sure no.
3. Can't tell anything really usefull about it. Except, i didn't hear such info in the past about my home ISP.

**Comment by morethanhater** [comment]
Additional info:
I found, that i can still use auth function at dummydroid even w/o license.
Indeed, it says that account is disabled. My phone is attached to this account and bound to two devices. I get auth request on my device which i need to approve. Then, i see endless loop.
These factors are frustrating, because such situation isn't described at your blog post. And, well... Hey, i can still log-in to account and use it, but it's disabled. Hecc?))

More:
I did change password, enabled two-factor and device auth methods but it still says disabled...

And new accounts says, that they are disabled. Guess, it's my ISP then.

Closed because of old repo.
